odoo/fields.py

```diff
@@ -363,7 +363,7 @@ def _get_attrs(self, model_class, name):
         attrs['model_name'] = model_class._name
         attrs['name'] = name
         attrs['_module'] = modules[-1] if modules else None
-        attrs['_modules'] = set(modules)
+        attrs['_modules'] = tuple(unique(modules))
```

Comment:
`unique` creates a set internally, since order was clearly irrelevant why not just wrap the `tuple` call around the `set`?